### ISSUE TITLE
How to add a second tv?

### ISSUE THREAD

**Issue by asbesto01** [AlexanderBabel/homebridge-broadlink-rm#82]
Hello,

i configured the config.json properly, in order to have two TVs. when I add the homebridge to my home app, it only discover one TV accessory.

How to solve this? I see in the guide:

You added more than TVs in your configuration!
Due to a HomeKit limitation you need to add any additional TV to the Home app by using the Add Accessory function.
There you'll find your additional TVs and you can use the same PIN as you using for this HomeBridge instance.

for this reason, I try to manually add the accessory within the home app, using the same PIN shared with the homebridge (880-75-559), but nothing is discovered! any hints?

This is my json thanks

{
 "bridge": {
 "name": "Homebridge XXX",
 "username": "XXXX",
 "port": XX,
 "pin": "880-75-559"
 },
 
 "description": "HomePi Homebridge.",
 
  "accessories":[
 {
  "accessory": "XiaomiRoborockVacuum",
 "name": "Aspirapolvere",
 "ip": "XXXX",
 "token": "XXXX",
 "pause": false,
 "dock": false
 }
  ],

  "platforms":[
    {
      "platform":"BroadlinkRM",
      "name":"Broadlink RM",
      "hideScanFrequencyButton": true,
      "hideLearnButton": true,
      "hideWelcomeMessage": false,
      "accessories":[
{

"name":"TV Salotto",
"type":"tv",
"host": "192.168.0.42",
          "data": {
            "off": "2600500000012894150f15111435151014111411141114101435153515101435153515341535153515101411141114351411141114111410143515351435151015351435153515351400052300012a4814000d050000000000000000",
            "on": "2600500000012993141114101535151014101510151015101534153515101534153515341535153514101510151015351410151015101510153415351534151015351435153515351400052300012a4814000d050000000000000000",
            "remote": {
              "select": "260050000001289313131312133614121312131213131312133614361312143613371336143613371312131213371312131214121336131313361436131213371337133613121436130005280001284a13000d050000000000000000",
              "arrowUp": "26004800000127941411121313371311151112121411131213371238121313361337123812371436141112131312141113111411123812131337123713371238143514371311143614000d05",
              "arrowDown": "26004800000128941510121313371410141112131411131213361337121313371238123712381435153514111313131114111213143613111510133712381237143614361411143515000d05",
              "arrowLeft": "2600480000012992170f121315341510160f12131510151015341238141214351534123814361534163415351535131116101114141015111410151013121534163412381435163416000d05",
              "arrowRight": "2600480000012992170f121315341510160f1213150f161015341238141115351435123814351634161011381534161012121511131214101633161013121534153514361435163415000d05",
              "back": "260048000001279415101313133614111411121215101313133612381312133713361238123713371411121314111435151112371213141114361435143614111436160f1237143614000d05",
              "exit": "26004800000127941411121413361411141112121411131313361238121313371336123812371436143614361311143614361312133615111212141113371411121215361311143614000d05",
              "playPause": "2600500000012794160f1312133713111312131213121213133614361312133614361336143613361436133713111337131213121311131213121312133614111337133614361336140005240001274b13000d050000000000000000",
              "info": "2600480000012a91170f13121435150f161012131410161014351435151115341238143615341634153515351410153515101535151014351411151015351510153415111435151015000d05"
            },
            "powerMode": {
              "show": "HEX..."
            },
            "volume": {
              "up": "26004e00939116131213133614351313113812121312131212131237153515101435160f1510153416341337150f1634160f1510121213121312121312371411153513361436130006029394143613000d0500000000000000000000",
              "down": "26004e00939413121311133713371311133713121312131113121337133614111337131113121312133713361411133713111312131213361411131213361411133713371336140006019494133614000d0500000000000000000000"
            },
            "inputs": [
              {
                "name": "Acceso/a",
                "type": "hdmi",
                "data": "HEX..."}]}},
{
"name":"TV Camera",
"type":"tv",
"host": "192.168.0.85",
          "data": {
            "off": "2600500000012894150f15111435151014111411141114101435153515101435153515341535153515101411141114351411141114111410143515351435151015351435153515351400052300012a4814000d050000000000000000",
            "on": "2600500000012993141114101535151014101510151015101534153515101534153515341535153514101510151015351410151015101510153415351534151015351435153515351400052300012a4814000d050000000000000000",
            "remote": {
              "select": "260050000001289313131312133614121312131213131312133614361312143613371336143613371312131213371312131214121336131313361436131213371337133613121436130005280001284a13000d050000000000000000",
              "arrowUp": "26004800000127941411121313371311151112121411131213371238121313361337123812371436141112131312141113111411123812131337123713371238143514371311143614000d05",
              "arrowDown": "26004800000128941510121313371410141112131411131213361337121313371238123712381435153514111313131114111213143613111510133712381237143614361411143515000d05",
              "arrowLeft": "2600480000012992170f121315341510160f12131510151015341238141214351534123814361534163415351535131116101114141015111410151013121534163412381435163416000d05",
              "arrowRight": "2600480000012992170f121315341510160f1213150f161015341238141115351435123814351634161011381534161012121511131214101633161013121534153514361435163415000d05",
              "back": "260048000001279415101313133614111411121215101313133612381312133713361238123713371411121314111435151112371213141114361435143614111436160f1237143614000d05",
              "exit": "26004800000127941411121413361411141112121411131313361238121313371336123812371436143614361311143614361312133615111212141113371411121215361311143614000d05",
              "playPause": "2600500000012794160f1312133713111312131213121213133614361312133614361336143613361436133713111337131213121311131213121312133614111337133614361336140005240001274b13000d050000000000000000",
              "info": "2600480000012a91170f13121435150f161012131410161014351435151115341238143615341634153515351410153515101535151014351411151015351510153415111435151015000d05"
            },
            "powerMode": {
              "show": "HEX..."
            },
            "volume": {
              "up": "26004e00939116131213133614351313113812121312131212131237153515101435160f1510153416341337150f1634160f1510121213121312121312371411153513361436130006029394143613000d0500000000000000000000",
              "down": "26004e00939413121311133713371311133713121312131113121337133614111337131113121312133713361411133713111312131213361411131213361411133713371336140006019494133614000d0500000000000000000000"
            },
            "inputs": [
              {
                "name": "Acceso/a",
                "type": "hdmi",
                "data": "HEX..."}]}}

]}

  ]
}

**Comment by sauriodino** [comment]
@asbesto01 Hi! Sorry for the question, is that “remote” configuration working as your remote in the appleTV from control center? Thanks in advance!

**Comment by anhtuanng98** [comment]
If you want to add the second TV, creat the second homebridge because remote app can work perfectly with only one TV per bridge. If you have 2 TVs on a bridge, only one appear at remote app.

However, I am not sure that new version of homebrige could fix it or not.

**Comment by asbesto01** [comment]
> @asbesto01 Hi! Sorry for the question, is that “remote” configuration working as your remote in the appleTV from control center? Thanks in advance!

Yes of course! ![image](https://user-images.githubusercontent.com/48592932/80442226-c7a03680-890c-11ea-8180-a7e76ea8199c.png)

**Comment by asbesto01** [comment]
> If you want to add the second TV, creat the second homebridge because remote app can work perfectly with only one TV per bridge. If you have 2 TVs on a bridge, only one appear at remote app.
> 
> 
> 
> However, I am not sure that new version of homebrige could fix it or not.

I don’t know how I solved it, but I managed to do that in one single home bridge:

![image](https://user-images.githubusercontent.com/48592932/80442333-f1f1f400-890c-11ea-99af-6cb294a22f56.png)

Running home bridge 0.4.53

With the first post Json

**Comment by anhtuanng98** [comment]
> > If you want to add the second TV, creat the second homebridge because remote app can work perfectly with only one TV per bridge. If you have 2 TVs on a bridge, only one appear at remote app.
> > However, I am not sure that new version of homebrige could fix it or not.
> 
> I don’t know how I solved it, but I managed to do that in one single home bridge:
> 
> ![image](https://user-images.githubusercontent.com/48592932/80442333-f1f1f400-890c-11ea-99af-6cb294a22f56.png)
> 
> Running home bridge 0.4.53
> 
> With the first post Json

But remote app where you can change the volumn, it can be control one TV per bridge.

**Comment by sauriodino** [comment]
@anhtuanng98 @asbesto01 
To add another tv maybe you can add it like this mock-up 

“Platform”: “rm-Broadlink”
[
{
“Name”: “Sony TV”,
Add all the frecuencies 
Remotes and inputs
Then },<<<Add a comma
{
“Name”: “Phillips TV”,
Frecuencies, remotes
And inputs}]
